hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -85,7 +88,7 @@ public void registerTask(ReconOmTask task) {
     taskFailureCounter.put(taskName, new AtomicInteger(0));
     // Create DB record for the task.
     ReconTaskStatus reconTaskStatusRecord = new ReconTaskStatus(taskName,
-        0L, 0L);
+        0L, 0L, null);
```

Comment:
```suggestion
        0L, 0L, 0);
```

Reply:
Addressed in [5248373](https://github.com/apache/ozone/pull/7517/commits/5248373fede0a47e1475ef14730e4b443445b55d)